script/sign-json.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
```

Comment:
Thoughts on perhaps moving this into Homebrew/brew?

Reply:
Yeh, we should move JSON generation to brew ASAP too IMO. Could write up a `help wanted` issue for that?

Reply:
I opened one here: https://github.com/Homebrew/brew/issues/14730. Feel free to modify if I missed anything

Reply:
Thanks!